contracts/core/Allo.sol

```diff
@@ -1,128 +1,70 @@
 import "@openzeppelin-upgradeable/token/ERC20/IERC20Upgradeable.sol";
-import "@openzeppelin-upgradeable/token/ERC20/utils/SafeERC20Upgradeable.sol";
-import {AccessControl} from "@openzeppelin/access/AccessControl.sol";
+import "@openzeppelin/access/AccessControl.sol";
 
 import "@solady/auth/Ownable.sol";
 
-import {Metadata} from "./libraries/Metadata.sol";
 import {Clone} from "./libraries/Clone.sol";
 import {Transfer} from "./libraries/Transfer.sol";
-import {IStrategy} from "../strategies/IStrategy.sol";
-import {Registry} from "./Registry.sol";
 
 /**
- *          ___           ___       ___       ___
- *         /\  \         /\__\     /\__\     /\  \
- *        /::\  \       /:/  /    /:/  /    /::\  \
- *       /:/\:\  \     /:/  /    /:/  /    /:/\:\  \
- *      /::\~\:\  \   /:/  /    /:/  /    /:/  \:\  \
- *     /:/\:\ \:\__\ /:/__/    /:/__/    /:/__/ \:\__\
- *     \/__\:\/:/  / \:\  \    \:\  \    \:\  \ /:/  /
- *          \::/  /   \:\  \    \:\  \    \:\  /:/  /
- *          /:/  /     \:\  \    \:\  \    \:\/:/  /
- *         /:/  /       \:\__\    \:\__\    \::/  /
- *         \/__/         \/__/     \/__/     \/__/
+ *          ___            ___        ___        ___
+ *         /\  \          /\__\      /\__\      /\  \
+ *        /::\  \        /:/  /     /:/  /     /::\  \
+ *       /:/\:\  \      /:/  /     /:/  /     /:/\:\  \
+ *      /::\~\:\  \    /:/  /     /:/  /     /:/  \:\  \
+ *     /:/\:\ \:\__\  /:/__/     /:/__/     /:/__/ \:\__\
+ *     \/__\:\/:/  /  \:\  \     \:\  \     \:\  \ /:/  /
+ *          \::/  /    \:\  \     \:\  \     \:\  /:/  /
+ *          /:/  /      \:\  \     \:\  \     \:\/:/  /
+ *         /:/  /        \:\__\     \:\__\     \::/  /
+ *         \/__/          \/__/      \/__/      \/__/
  */
 
 /// @title Allo
 /// @notice The Allo contract
 /// @author allo-team
-contract Allo is Transfer, Initializable, Ownable, AccessControl {
-    using SafeERC20Upgradeable for IERC20Upgradeable;
-
-    /// @notice Custom errors
-    error UNAUTHORIZED();
-    error NOT_ENOUGH_FUNDS();
-    error NOT_APPROVED_STRATEGY();
-    error IS_APPROVED_STRATEGY();
-    error MISMATCH();
-    error ZERO_ADDRESS();
-    error INVALID_FEE();
-
-    /// @notice Struct to hold details of an Pool
-    struct Pool {
-        bytes32 identityId;
-        IStrategy strategy;
-        address token;
-        uint256 amount;
-        Metadata metadata;
-        bytes32 managerRole;
-        bytes32 adminRole;
-    }
-
+contract Allo is IAllo, Transfer, Initializable, Ownable, AccessControl {
     /// @notice Fee denominator
     uint256 public constant FEE_DENOMINATOR = 1e18;
 
     /// ==========================
     /// === Storage Variables ====
     /// ==========================
 
+    address constant NATIVE = address(0);
     /// @notice Fee percentage
     /// @dev 1e18 = 100%, 1e17 = 10%, 1e16 = 1%, 1e15 = 0.1%
-    uint256 public feePercentage;
+    uint256 private feePercentage;
```

Comment:
Did we decide not to make our private variables like this `_feePercentage`? Or are we just doing this for internal functions?

Reply:
i think currently only for internal functions, or i miss something :D